packages/semaphore-identity/package.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "@zk-kit/semaphore-identity-artifacts",
+    "description": "semaphore identity artifacts",
+    "license": "AGPL-3.0",
```

As discuss privately let's change to
```suggestion
    "license": "MIT",
```
?

---

Yess!